super_csv/serializers.py

```diff
@@ -38,5 +44,22 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        """
+        Get data
+        """
+        try:
+            data = json.load(operation.data)
+        except (FileNotFoundError, ValueError):
+            msg = _('Failed to retrieve file.')
+
+            logger.error('Failed to retrieve operation data for operation %s course %s',
```

Comment:
please put this string, each argument, and the closing paren on their own lines 

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization